backend/src/rules/abilities.clj

```diff
@@ -3,16 +3,37 @@
 (defn add-power
   "Creates the ability function"
   [increase]
-  (fn    
-;   "Alters a cards' power, by adding the a defined value to it.
-;   This power may be negative, resulting in a decrease"
-    ([game-state target]
+  {:ability
+   (fn    
+;    "Alters a cards' power, by adding the a defined value to it.
+;    This power may be negative, resulting in a decrease"
+     [game-state play]
      (update-in
        game-state
-       [:cards target :power]
+       [:cards (:target play) :power]
        #(+ % increase)))
-     ([]
-      {:description (if (neg? increase)
-                      (str "Weaken " (- increase))
-                      (str "Enhance " increase))
-       :target 1})))
+   :description (if (neg? increase)
+                    (str "Weaken " (- increase))
+                    (str "Enhance " increase))
+   :target 1})
+
+(defn type-add-power
```

Comment:
I dislike the fact that the name of the function is so similar to `add-power`... I think at least a reference to the row is missing, like `add-power-on-row-type` or something